_docs/using/configuration.md

```diff
@@ -8,6 +8,11 @@ Configuring the bot is relatively straight forward. First, open your bot folder
 
 > Editing your configuration file using Notepad or WordPad will result in problems as they save files in strange formats and aren't designed for editing .ini files.
 
+#### Intents
```

Comment:
This section can be moved to troubleshooting, along with the information about this issue.

Reply:
Done.